contracts/zero-ex/contracts/src/external/TransformerDeployer.sol

```diff
@@ -22,64 +22,96 @@ pragma experimental ABIEncoderV2;
 import "@0x/contracts-utils/contracts/src/v06/AuthorizableV06.sol";
```

Comment:
Let's put this in a new file, something like `PermissionlessTransformerDeployer`. We will likely continue using the existing `TransformerDeployer` for some time.

Reply:
There are lots of test changes that need to go along with switching the deployer because we move from nonces to init code hashes and salts. I think those are hard to keep side-by-side, so I'll try to figure out a way to preserve those. Maybe I'll just leave this branch with all the test changes, and we can just never merge it. But at least we'll have those changes handy when it's time for someone to recreate them.

Then I can just make a new PR that just adds `PermissionlessTransformerDeployer` and unit tests just its functionality.

What do you think?

Reply:
It's not just tests, actually. `TransformERC20Feature` has to change, etc. So there's no really good side-by-side version of this, but we can still park just the `PermissionlessTransformerDeployer`. We just need to be aware that when it's time to use it, there will be a cascade of other code changes required.

Reply:
Oh yeah, I would be in favor of splitting into two PR's. We can merge and audit the Solidity next week, then keep an open WIP PR w/ the integrations and testing. We can build off the testing PR when we're ready to make the switch.